File: source/code/vbnet/authentication/get_asm_req.vb
Dim app = Await client.GetApplicationAsync(
    "https://api.stormpath.com/v1/applications/5nan67mWrYrBmLGu7nGurh")

Dim accountStoreMappings = Await app _
    .GetAccountStoreMappings() _
    .ToListAsync()

File: source/code/java/authentication/get_asm_req.java
import com.stormpath.sdk.application;
import com.stormpath.sdk.client;

Application application = client.getResource("https://api.stormpath.com/v1/applications/5nan67mWrYrBmLGu7nGurh", Application.class);

ApplicationAccountStoreMappingList application.getAccountStoreMappings();
